### ISSUE TITLE
Motion to Vacate | Granted | Update attorney review screen

### ISSUE THREAD

**Issue by jimruggiero** [department-of-veterans-affairs/caseflow#13007]
As a decision drafting attorney I want to see a screen that allows me to review granted motions to vacate from a judge. 

## Acceptance criteria
- [ ] When the appeal stream_type is Vacate, after a decision attorney selects the "Decision ready for review" action on their AttorneyTask, they should be directed to `/appeals/[uuid]/tasks/[id]/draft_decision/vacatures` (ultimately up to the engineer)
  - Note: This page will use the same component as the `draft_decision/dispositions` page.

When the URL matches that pattern, the component implements the following changes:
- [ ] Update header copy on the "Draft decisions" screen to "Review vacated decision issues"
- [ ] Update the copy for the text below the header (see Copy)
- [ ] Hide the "Add decision" button
- [ ] Hide "Delete" for now (added to scope afterwards, differs from screenshot)

## Copy
Header: Review vacated decision issues
Text: Review the prior decision issues here that have been marked for vacatur by the Judge. Please remember to include them in your decision letter if you haven't already.

### Out of scope
- "return to judge" banner
- Top bar showing steps 1 and 2 (this varies per vacate type)

### Designs
https://app.mural.co/t/workqueue2001/m/workqueue2001/1573574515518/2c771474e7259260c0ebc68b44c78bb98f295b04

![b872c96b-7f39-4c0c-8df1-5d15f76f2d0a.png](https://images.zenhubusercontent.com/58f6b6eb51aa2c30577bb60e/0b032e49-7d8c-4e4d-9995-29f4d936b631)

### Tech notes
This is happening on the AttorneyTask, which is a child of JudgeDecisionReviewTask
The Draft Decisions URL that this page shows up on:
`/queue/appeals/887309dd-2bda-4d38-b4e9-7545099b34b3/tasks/9/draft_decision/dispositions`

Original copy of text:
"Add decisions to each issue the Veteran is appealing. Any decisions you add must be connected to at least one issue on appeal. This helps Caseflow gather metrics on BVA decisions and meet AMA reporting requirements."

### Open Questions
N/A

### Resources/other links
N/A

**Comment by jimruggiero** [comment]
![image (9).png](https://images.zenhubusercontent.com/5d2748c531297d7d65f8d26b/b872c96b-7f39-4c0c-8df1-5d15f76f2d0a)

**Comment by jcq** [comment]
Existing flows use `SelectDispositionsView` within `SelectDispositionsContainer`. Currently copy (which would be different in these designs) is specified directly in `SelectDispositionsView`, but that could be altered to accept props.

**Comment by jcq** [comment]
@nanotone In order to (ever-so-slightly) reduce clutter in `QueueApp`, I propose we have a single route that handles both this and #13009. We can create a container component that itself includes the two sub/child routes. Does that work with your thinking?

**Comment by leikkisa** [comment]
In light of the changes re: appeal streams, we won't be building a new screen for this anymore. I'm going to keep this ticket open temporarily until there's a ticket to account for the modifications to the existing screen for an attorney to review decisions.

**Comment by leikkisa** [comment]
This ticket has been updated from the original scope of the ticket

**Comment by jcq** [comment]
@leikkisa If this screen is for the attorney to simply review/verify that dispositions are already set for the issues the judge selected, what is the behavior of the `edit` button? And if the user (atty) deletes (removes) an issue, what happens? Is it sufficient to keep track of the issues that remain, send those as part of our payload in the future step, and update the decision IDs on `PostDecisionMotion` at that point? Do we need any sort of "undo" functionality, or if the attorney removes/deletes one that they shouldn't, do we just say they need to start over w/ this task?

**Comment by leikkisa** [comment]
@jcq

We want to hide the "delete" button. I hadn't yet updated the screenshot, but added it to the AC.  I feel like we could hide the "edit" button here too, but I think in conversation with design, we decided to keep it in case the decision text needed to be edited.

I haven't confirmed at what point things get saved in the database, if that happens within different steps of the task, or not until it is submitted.  The judge will have the ability to review and edit the issues though, so the attorney's choices here are undoable.  There might be a way to return to the attorney too in case the Judge wants the attorney to make changes - not sure at the moment.

Validation of the issues is something that we have noted for working on after the MVP. This would be making sure the vacated issues all have a vacated decision issue, and readjudicated issues have that plus a new disposition.

I guess the short answer is to move forward for now, and we'll refine and iterate on it after the MVP is done, including thinking more about "undos" and issue structure validation.
